### PR TITLE
cloud page created

### PR DESCRIPTION
**Pull request by HM9N**
Después de ensayar varias versiones, funciona correctamente con Node v12.16.1.

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -161,6 +161,7 @@ collections:
                     fields:
                       [
                         { label: Image, name: image, widget: image },
+                        { label: title, name: title, widget: text },
```
Comment (jmedina2930) [inline static/admin/config.yml]: Label should be with Capital Case, shouldn't it?

File: package.json [hunk 2]
```diff
@@ -45,6 +45,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "prettier": "^1.15.3"
+    "autoprefixer": "^10.4.0",
+    "gatsby-plugin-postcss": "^5.2.0",
```
Comment (jmedina2930) [inline package.json:50]: What are you using postcss plugin for? I thought you were just using tailwind.

**Review by jmedina2930 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (HM9N) [reply package.json:50]: Now I realize that I'm using gatsby-plugin-postcss to give the style i want to. Both work together (TailwindCSS and gatsby-plugin-postcss).  In the postcss.config.js file you can see that i'm using tailwindcss as a postcss plugin.

File: static/admin/config.yml (hunk 1 above)
Comment (HM9N) [reply static/admin/config.yml]: Yes. Fixed it.

File: package.json (hunk 2 above)
Comment (HM9N) [reply package.json:50]: I'm not using "prettier" anymore. I need to use "autoprefixer" to parse css and add css vendor prefixes, this is necessary to use the postcss plugin.

**Review by jmedina2930 — APPROVED** [review]

**Pull request merged**